### PR TITLE
[FIX] mail: preview of some documents were broken

### PR DESCRIPTION
**Pull request by adsh-odoo**
**Current behavior before PR:**

Preview of some PDF documents were broking.

**Desired behavior after PR is merged:**

All PDF documents are previewable now.

Task-3305415

### CODE REVIEW

File: addons/mail/static/src/attachments/attachment_model.js [hunk 1]
```diff
@@ -52,7 +52,8 @@ export class Attachment {
     }
 
     get isPdf() {
-        return this.mimetype === "application/pdf";
+        const mimetype = ["application/pdf;base64", "application/pdf" ]
+        return mimetype.includes(this.mimetype)
```
Comment (std-odoo) [inline addons/mail/static/src/attachments/attachment_model.js:56]: can we do `mimetype.startsWith("application/pdf")` instead ?

File: addons/mail/static/src/attachments/attachment_model.js [hunk 2]
```diff
@@ -52,7 +52,8 @@ export class Attachment {
     }
 
     get isPdf() {
-        return this.mimetype === "application/pdf";
+        const mimetype = ["application/pdf;base64", "application/pdf" ]
```
Comment (std-odoo) [inline addons/mail/static/src/attachments/attachment_model.js:55]: can you elaborate a bit more the issue in your commit message ?

**Review by std-odoo — COMMENTED** [review]
@adsh-odoo Hi :) 

I have a few comments

Cheers

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**